shell/run_cmd.go

```
@@ -69,6 +69,16 @@ func RunCommandWithOutput(
 			cmdStdout = io.MultiWriter(opts.Writer, &output.Stdout)
 		)
 
+		// Pass the traceparent to the child process if it is available in the context.
+		traceParent, err := telemetry.TraceParentFromContext(ctx)
+		if err != nil {
+			return errors.New(err)
+		}
+		if traceParent != "" {
```

golangci-lint (1.64.8)

if statements should only be cuddled with assignments

(wsl)